content/setting-up-tugboat/select-repo-settings.md

```diff
@@ -127,6 +127,18 @@ credentials from within the repo settings.
 
 ![Authenticate with a Docker registry](../../_images/authenticate-with-a-docker-registry-add-credentials.png)
 
+By default, when you
+[specify a public image to use in Tugboat builds](/setting-up-services/how-to-set-up-services/specify-a-service-image),
+we use a Tugboat-authenticated user to pull images from Docker Hub.
+
+If you need to pull an image that requires authentication, adding your Docker Registry Authentication credentials to the
+repository settings will use those credentials for every Docker pull made from this repository, overriding Tugboat's
+default user.
+
+By managing these credentials on the repository-level, there's no need to specify credentials in your Tugboat
+`config.yml` file. Setting those credentials here automatically applies them
+[when Tugboat pulls images](/setting-up-services/service-images/docker-pull/) to build your previews.
```

Comment:
It might be worth pointing out that in order to set dockerhub credentials, the `Server` field should be left blank.

Also, probably worth noting, is that multiple authentications can be specified for the same registry, and Tugboat will try to use each one to pull an image until one works, or will throw an error if none of them work.